routes/index.js

```diff
@@ -356,6 +357,19 @@ router.post('/register-admin', (req, res) => {
     // let user know that access is denied, do not let them know there is a host requirement
     res.status(500).json("access denied");
   }
-})
+});
+
+/* GET uuid-gen - generate a uuid, store it in list, send it back to user */
+router.get('/uuid-gen', (req, res) => {
+  var uuidStr = uuid.GenerateUUID();
+  uuid.addUUIDToList(uuidStr);
+  req.flash('success', uuidStr);
+  res.redirect('/new-job');
+});
+
+/* GET new-job - visit new job page */
+router.get('/new-job', (req, res) => {
```

Comment:
How is the UUID used in `/new-job`?

Reply:
the page new-job has a button to generate uuid, returned in a flash when it gets from /uuid-gen (which will redirect to /new-job)

Reply:
Oo i see, so this is the new job creation page